ngraph/frontend/onnx_editor/src/detail/subgraph_extraction.cpp

```diff
@@ -99,20 +99,49 @@ namespace
         return *it;
     }
 
+    std::string get_input_tensor_name(ONNX_NAMESPACE::GraphProto& graph, const InputEdge& edge)
+    {
+        validate_node_index(graph, edge.m_node_idx);
+        NGRAPH_CHECK(edge.m_port_idx >= 0 &&
+                         edge.m_port_idx <
+                             static_cast<int>(graph.node(edge.m_node_idx).input().size()),
+                     "Node with index: ",
+                     std::to_string(edge.m_node_idx),
+                     " has not input with index: ",
+                     std::to_string(edge.m_port_idx));
```

Comment:
Could you add a helper for validating ports?
It seems odd that node_index have a helper for it but port doesn't.

Reply:
done